ArrayUtils.js

```diff
@@ -1,21 +1,65 @@
-class ArrayUtils{
+class ArrayUtils {
 
-    isEmpty(array){return false;}
+    constructor(original, array, value) {
+        this.original = original;
+        this.array = array;
+        this.value = value;
+    }
 
-    append(original, value){return original;}
+    isEmpty(array) {
+        if (array[""]) {
+            return false;
+        } else {
+            return true;
+        }
+    }
 
-    clone(original){return original;}
+    append(original, value) {
+        return original.push(value);
+    }
 
-    subArray(original, from, to){return original;}
+    getClone(original) {
+        clone = [...original];
+        return clone;
+    }
 
-    equals(arr1, arr2){return false;}
+    getSubArray(original, y, z) {
+        let subArray = original.slice(y, z + 1);
+        return subArray;
+    }
 
-    fill(original, value){}
+    arraysEqual(arr1, arr2) {
+        if (arr1 === arr2)
+            return true;
+        else if (arr1 == null || arr2 == null)
+            return false;
+        else(arr1.length !== arr2.length)
+        return false;
+    }
 
-    indexOf(original, value){return -1;}
+    fillArray(original, value) {
```

Comment:
`indexOf()` and `remove()` work, make sure you know how to do them manually as well